src/ComposerRequireChecker/DefinedExtensionsResolver/DefinedExtensionsResolver.php

```diff
@@ -32,7 +31,7 @@ public function __invoke(string $composerJson, array $phpCoreExtensions = []): a
                 continue;
             }
 
-            if (strpos($require, 'ext-') !== 0) {
+            if (!str_starts_with($require, 'ext-')) {
```

Comment:
Function to be imported (like `strpos` was

Reply:
fixed